src/data/agile-quiz.ts

```diff
@@ -1,4 +1,13 @@
 const agileQuiz = [
+  {
+    Question: "When might a Sprint be abnormally cancelled?",
+    Answer: "When the Sprint Goal becomes obsolete.",
+    Distractor1: "When it becomes clear that not everything will be finished by the end of the Sprint.",
+    Distractor2: "When the Developers feel that the work is too hard.",
+    Distractor3: "When the sales department has an important new opportunity.",
+    Explanation: "A Sprint could be cancelled if the Sprint Goal becomes obsolete. Only the Product Owner has the authority to cancel the Sprint.",
```

Comment:
Please avoid using direct quotes from 3rd party resources. It is always best to rewrite it in your own words. 

```suggestion
    Explanation: "The Product Owner can cancel a Sprint if the Sprint Goal becomes obsolete.",
```

Reply:
I see. Thanks for the advice.